camptix.php

```diff
@@ -6591,6 +6584,7 @@ function review_timeout_payments() {
 	 * Step 3: Uses a payment method to perform a checkout.
 	 */
 	function form_checkout() {
+
```

Comment:
I think there a few minor PHPCS violations in the PR. Since it's such a big file w/ so many other violations, it might be helpful to use `wp-dev-lib`, so that you can run PHPCS on only the _changed_ lines. There's more info on that here:

https://make.wordpress.org/core/handbook/best-practices/commit-messages/#other-tips-for-committers